client/task-list/tasks/payments/components/PaymentMethodList/PaymentMethodList.scss

```diff
@@ -0,0 +1,106 @@
+.woocommerce-task-payment {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	position: relative;
+	overflow: hidden;
+
+	.woocommerce-task-payment__recommended-ribbon {
+		position: absolute;
+		transform: rotate(-45deg) translate(-50%, -50%);
+		background: $studio-gray-80;
+		color: $studio-white;
+		font-size: 11px;
+		line-height: 20px;
+		position: absolute;
+		top: 0;
+		left: 0;
+		line-height: 1;
+		padding: 7px $gap-largest;
+		transform-origin: top left;
+		margin-top: 32px;
+		margin-left: 32px;
+
+		span {
+			max-width: 70px;
+		}
+	}
+
+	.components-card__media {
+		width: 170px;
+		flex-shrink: 0;
+
+		img,
+		svg {
+			margin: auto;
+			max-width: 100px;
+			display: block;
+		}
+	}
+
+	> .components-form-toggle {
+		min-width: 52px;
+	}
+
+	.woocommerce-task-payment__title {
+		font-size: 16px;
+		font-weight: 500;
+		color: $studio-gray-80;
+		margin-top: 0;
+		margin-bottom: $gap-smaller;
+	}
+
+	.woocommerce-task-payment__content {
+		font-size: 14px;
+		color: $studio-gray-60;
+		margin: 0;
+		margin-right: $gap-larger;
+
+		.text-style-strong {
+			font-weight: bold;
+		}
+
+		p {
+			font-size: 12px;
+		}
+	}
+	.woocommerce-task-payment__description {
+		flex: 1;
+	}
+
+	@include breakpoint( '<600px' ) {
+		flex-wrap: wrap;
+
+		.woocommerce-task-payment__recommended-ribbon {
+			margin-top: 24px;
+			margin-left: 24px;
+			font-size: 9px;
+		}
+
+		.woocommerce-task-payment__content {
+			margin: 0;
+		}
+
+		.components-card__media {
+			> svg,
+			> img {
+				margin: 0 0 0 $gap-large;
+			}
+
+			order: 1;
+			flex-basis: 50%;
+		}
+
+		.woocommerce-task-payment__description {
+			order: 3;
+			padding: $gap-large 0 0 $gap-large;
+		}
+
+		.woocommerce-task-payment__footer {
+			flex-basis: 50%;
+			align-self: flex-end;
```

Comment:
I feel like on smaller screens this should be flex-start or center.  What do you think?

Current:
<img width="414" alt="Screen Shot 2021-04-15 at 12 11 14 PM" src="https://user-images.githubusercontent.com/10561050/114902434-d44ec080-9de3-11eb-81fc-b2dc03957ad7.png">


Start:
<img width="418" alt="Screen Shot 2021-04-15 at 12 11 32 PM" src="https://user-images.githubusercontent.com/10561050/114902427-d153d000-9de3-11eb-944c-74088af9c44c.png">

Reply:
I think I like it. Done in bfbec15cbfc2d27f5732c60879b7d02145c3e720